### ISSUE TITLE
Cannot add a custom picture theme for new tab

### ISSUE THREAD

**Issue by Titi68730** [Alex313031/thorium#845]
**System Details**
 - OS: Windows 11
 - Thorium Version 126.0.6478.251 (64 bits)

**Problem**
I can't change the background picture. When I upload it, it's showing for like 2-3 seconds and all of a sudden, it disappears and my thorium colors changed based on the picture color.
**Did you read the FAQ?**
Yes I did

![image](https://github.com/user-attachments/assets/97a13561-23b7-4823-ae93-7c7e874bc66f)
Expected background
![zazu wallpaper](https://github.com/user-attachments/assets/83899ab1-6423-4c93-b20c-02aa4db06615)

**Comment by gz83** [comment]
Perhaps you could try version 150 if you're still using thorium.

**Comment by mokotanin** [comment]
yeah yeah it works now 😳 u can close this issue (old asf too, almost two years old)
